tools/test.py

```diff
@@ -875,6 +874,11 @@ def GetVmFlags(self, testcase, mode):
   def GetTimeout(self, mode):
     return self.timeout * TIMEOUT_SCALEFACTOR[ARCH_GUESS or 'ia32'][mode]
 
+  def GetEngine(self, arch, mode):
```

Comment:
> may be we could port this to chakracore-master as well.

Yes. (The reason I changed this is because we don't have an external passed in "engine = ..." switch.)